client/my-sites/stats/mobile-promo-cards/index.jsx

```diff
@@ -2,22 +2,22 @@ import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { MobilePromoCard } from '@automattic/components';
 import { translate } from 'i18n-calypso';
 import { useEffect } from 'react';
-import wordpressSeoIllustration from 'calypso/assets/images/illustrations/wordpress-seo-premium.svg';
+import blazeDropDownIllustration from 'calypso/assets/images/illustrations/blaze-drop-down.svg';
```

Comment:
Just repeating myself but maybe we keep both?

Reply:
Good suggestion. We decided to maintain both banners prioritizing Blaze